frontend/modules/shared/ui/src/lib/layout/language-switcher/language-switcher.ts

```diff
@@ -0,0 +1,48 @@
+import { CommonModule } from '@angular/common';
```

Comment:
There's a high possibility that this language-switcher component should be somewhere else but i dont know where

Reply:
I think you can move it to the root project (src/app) since it's not gonna be used elsewhere and ngx-translate configuration also is there

Reply:
Nevermind, it's used in top-bar component so it must be available for it.